demos/common/include/iot_config_common.h

```diff
@@ -155,6 +156,16 @@
     #define IOT_NETWORK_RECEIVE_TASK_STACK_SIZE    ( 5 * configMINIMAL_STACK_SIZE )
 #endif
 
+/* Default iot thread stack_size. */
+#ifndef IOT_THREAD_DEFAULT_STACK_SIZE
+    #define IOT_THREAD_DEFAULT_STACK_SIZE    configMINIMAL_STACK_SIZE
+#endif
+
+/* Default iot thread priority. */
+#ifndef IOT_THREAD_DEFAULT_PRIORITY
+    #define IOT_THREAD_DEFAULT_PRIORITY      5
```

Comment:
Why 5?  Also it's not guaranteed that some boards have 5 discrete priorities.

We generally default to idle priority.

Reply:
Again, I defaulted this to what the POSIX code was doing. This is a config param so you can make it what you need to.

Reply:
I updated this one to tskIDLE_PRIORITY as per your request, when a platform needs this to be higher it can be overriden easy enough.